etc/p3-config/p3-config.yml

```diff
@@ -367,6 +377,8 @@ p3_flavor_compute_A_magnum:
   disk: 271
   vcpus: 64
   ephemeral: 100
+  extra_specs:
+    "resources:CUSTOM_COMPUTE_A": 1
```

Comment:
Will the ephemeral disk still be passed through?

Reply:
I can't remember if you followed up about this.

Reply:
so it gets passed through in the same way, I think its all in that patch call you just changed.